test/controllers/project_controller_test.exs

```diff
@@ -30,24 +21,12 @@ defmodule CodeCorps.ProjectControllerTest do
       project_1 = insert(:project, title: "Test Project 1", organization: organization)
       project_2 = insert(:project, title: "Test Project 2", organization: organization)
```

Comment:
Are the titles here necessary? Thoughts on reducing this to `insert_pair` and letting the factory generate the titles?  Since we aren't testing for them, I'm not sure we need them to be unique.